packages/jellyfish-transaction/__tests__/script/index.test.ts

```diff
@@ -954,6 +954,42 @@ describe('[OP_XOR]', () => {
   })
 })
 
+describe('[OP_EQUAL]', () => {
```

Comment:
I added missing OP_EQUAL and OP_EQUALVERIFY just now.

Reply:
thanks!